### PR TITLE
Add EC recovery instructions

### PR DESCRIPTION
**Pull request by mkopec**
Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: docs/variants/novacustom_ns5x/ec-recovery.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+- Build the firmware for your laptop:
+
+<!--
+  TBD: Device specific instructions, I guess we should add our own mainboards
+  in a different folder in the repository? For now let's just do system76/darp7
+-->
+
+```bash
+make BOARD=system76/darp7
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/ec-recovery.md:60]: @mkopec are we changing it now somehow?

File: docs/variants/novacustom_ns5x/ec-recovery.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  on both ends of the cable)
+- a 24-pin FFC breakout board with a 1.0mm pitch FFC connector and a 2.54mm
+  pitch pin header
+- A set of male-to-female jumper wires to connect the breakout to the Arduino
+- USB-A to USB-B cable to connect the Arduino to the host
+- USB-C cable for grouding
+
+<!--
+  The instructions assume pre-flashed Arduino at this point, so as to not
+  overcomplicate this instruction. Depending on wiring, the code (GPIO map) will
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/ec-recovery.md:25]: We should recommend only one wiring method. The simples one you suggested. Do we have the code for this wiring already in place?

File: docs/variants/novacustom_ns5x/ec-recovery.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+The firmware should now be built.
+
+## Assembly
+
+- Unscrew the bottom cover from the laptop
+- Disconnect the battery
+
+> All power must be removed from the laptop during flashing.
+
+- Reattach the bottom cover (without screwing it in) and flip the laptop over
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/ec-recovery.md:72]: Any photos?

File: docs/variants/novacustom_ns5x/ec-recovery.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/ec-recovery.md:60]: No, for now it remains unchanged. The slp_s0 fix is outside of the board directories

File: docs/variants/novacustom_ns5x/ec-recovery.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/ec-recovery.md:25]: I'll commit it to Dasharo/ec

Comment (mkopec) [reply docs/variants/novacustom_ns5x/ec-recovery.md:25]: Added: https://github.com/Dasharo/ec/pull/1

**Review by macpijan — APPROVED** [review]

**Pull request merged**